static/css/shared/_metadata_modal.scss

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Styling for chart metadata modal.
+ */
+
+@use "./modal.scss" as modal;
+
+
+.metadata-modal {
+  // TODO(beets): Apply these styles to all modals.
+  .modal-content {
+    border-radius: 28px;
+  }
+  .modal-header {
+    border-bottom: none;
+    padding: 24px 24px 8px 24px;
+  }
+  .modal-body {
+    padding: 8px 24px;
+  }
+  .modal-footer {
+    border-top: none;
+    padding: 24px;
+  }
+  .modal-title {
+    color: var(--gm-3-sys-light-on-surface);
+
+    /* title/medium */
+    font-family: "Google Sans Text";
+    font-size: 1.1rem;
+    font-weight: 500;
+    line-height: 1.5;
+  }
+
+  .modal-subtitle {
+    color: var(--gm-3-sys-light-on-surface);
+    padding: 0 24px 12px;
+
+    /* body/medium */
+    font-family: "Google Sans Text";
```

Comment:
I needed an override for Google Sans Text (rather than the current default Google Sans). I'll update to $headings-font-family for now. Perhaps either of those should be the default sans-serif vs headings font as a follow-on update.

Reply:
Good call-right now we have both "Google Sans", "Google Sans Text" in $font-family-sans-serif  - we should probably just have Google Sans Text in `$font-family-sans-serif` and Google Sans in `$headings-font-family`

Reply:
sounds good - will do this in a follow up